.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,16 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone:
+    name: Flowzone
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@ab77/operational
```

Comment:
don't merge until https://github.com/product-os/flowzone/pull/278 is merged and reset back to @master.